flink-ecosystem-backend/src/main/java/org/apache/flink/community/ReleaseController.java

```diff
@@ -31,15 +31,15 @@
 import java.util.List;
 
 @RestController
-@RequestMapping(value = "/release")
+@RequestMapping(value = "api/v1/release")
 public class ReleaseController {
```

Comment:
What's the purpose of a standalone release controller?

Reply:
just leave it as it was before, will merge to `FCPackageController`